ww-config.js

```diff
@@ -1,639 +1,745 @@
+    const data = wwLib.wwCollection.getCollectionData(content[basePropertyKey]);
+    if (!data.length || typeof data[0] !== 'object') {
+        return null;
+    }
+
+    return { object: data[0] };
+}
+
+export default {
+    editor: {
+        label: {
+            en: 'Calendar',
+            fr: 'Calendrier',
+        },
+        customStylePropertiesOrder: [
+            'defaultEventColor',
+            'defaultEventTextColor',
+            'defaultEventTitleSize',
+            'defaultEventTimeSize',
+            'defaultEventContentSize',
+            'themeColor',
+            ['themeMenuColor', 'themeTitleBarColor', 'themeTodayCellColor', 'themeSelectedCellColor'],
+            'defaultView',
+            ['enableYearsView', 'enableYearView', 'enableMonthView', 'enableWeekView', 'enableDayView'],
+            'enableTimelessMode',
+            ['timestep', 'timeStart', 'timeEnd'], // Steps control
+            ['startWeekOnSunday', 'twelveHour'], // I18n
+            ['hideWeekends', 'hideWeekdays'], // Hide days
+            'showAllDayEvents'
+        ],
+        customSettingsPropertiesOrder: [
+            'lang',
+            'enableMultiCalendar',
+            'calendars',
+            ['calendarIdPath', 'calendarLabelPath', 'calendarColorPath'],
+            'categories',
+            ['categoryNamePath', 'categoryColorPath', 'categoryColorTextPath'],
+            'events',
+            [
+                'eventStartPath',
+                'eventEndPath',
+                'eventTitlePath',
+                'eventContentPath',
+                'eventAllDayPath',
+                'eventCalendarPath',
+                'eventCategoryPath',
+            ],
+        ],
+    },
+    triggerEvents: [
+        {
+            name: 'event:click',
+            label: { en: 'On event click' },
+            event: {
+                rawEventData: {},
+                event: {
+                    start: '',
+                    end: '',
+                    title: '',
+                    content: '',
+                    calendar: null,
+                    allDay: null,
+                },
+                currentView: 'years | year | month | week | day',
+                domEvent: {},
+            },
+            default: true,
+            getTestEvent: 'getTestEvent',
+        },
+        {
+            name: 'cell:click',
+            label: { en: 'On cell click' },
+            event: {
+                cell: {
+                    date: new Date(),
+                    calendar: null,
+                },
+                currentView: 'years | year | month | week | day',
+            },
+        },
+        {
+            name: 'view:change',
+            label: { en: 'On view change' },
+            event: {
+                // TODO Check, not sure what should be defined here, see handleViewChange
+                currentView: 'years | year | month | week | day',
```

Comment:
You can put a sample of the event here so : 
```

{
     view: string
     startDate: null.
     endDate: null,
     firstCellDate: null,
     lastCellDate: null,
     outOfScopeEvents: [],
     events: [],
     week: 1
}
```
It will be displayed in the explorer when you bind in a workflow triggered by this event

You can remove "currentView" from here and from the trigger event call as the information can alrady be found in the "view" property

Reply:
Got it, makes sense.